src/desktop/apps/artist/server.js

```diff
@@ -56,10 +62,10 @@ app.get("/artist/:artistID*", async (req, res, next) => {
       },
       blocks: {
         head: () => (
-          <>
+          <React.Fragment>
             {headTags}
             <Meta sd={res.locals.sd} artist={artist} />
-          </>
+          </React.Fragment>
```

Comment:
Did this for now because our tslint setup was complaining about the `<>…</>` syntax.